pkg/manager/internal.go

```diff
@@ -454,11 +453,14 @@ func (cm *controllerManager) Start(stop <-chan struct{}) error {
 		go cm.serveHealthProbes(cm.internalStop)
 	}
 
-	go cm.startNonLeaderElectionRunnables()
+	doneCh := make(chan error, 1)
```

Comment:
Since we never put any errors on this, the convention within this code seems to be to use a `chan struct{}` as a signal channel, can we modify the PR to use that instead?